_data/sites.json

```diff
@@ -6154,16 +5984,6 @@
         ]
     },
 
-    {
-        "name": "Riffle",
-        "url": "https://riffle.uservoice.com/knowledgebase/articles/244346-delete-your-account",
```

Comment:
This URL is gone, but Riffle still exists and has an account deactivation/deletion process here: https://help.realnames.com/hc/en-us/articles/202814089-Deactivate-your-Email-Address